hunting/aws/docs/ssm_start_remote_session_to_ec2_instance.md

```diff
@@ -0,0 +1,1137 @@
+# SSM Start Remote Session to EC2 Instance
+
+---
+
+## Metadata
+
+- **Author:** Elastic
+- **Description:** This hunting query identifies when a user starts a remote session to an EC2 instance using the AWS Systems Manager (SSM) service. The `StartSession` API call allows users to connect to an EC2 instance using the SSM service. Multiple `StartSession` requests to the same EC2 instance may indicate an adversary attempting to gain access to the instance for malicious purposes. By default on certain AMI types, the SSM agent is pre-installed and running, allowing for easy access to the instance without the need for SSH or RDP.
+
+- **UUID:** `f9eae44e-5e4d-11ef-878f-f661ea17fbce`
+- **Integration:** [aws.cloudtrail](https://docs.elastic.co/integrations/aws/cloudtrail)
+- **Language:** `[ES|QL]`
+- **Source File:** [SSM Start Remote Session to EC2 Instance](../queries/ssm_start_remote_session_to_ec2_instance.toml)
+
+## Query
```

Comment:
looks like the sql query is missing here

Reply:
Fixed!